docs/theory/force_calibration/active.rst

```diff
@@ -91,3 +91,94 @@ hydrodynamically correct model for the peak:
 We can also include a distance to the surface like before. This results in an expression for the drag
 coefficient :math:`\gamma` that depends on the distance to the surface which is given by the same
 equations as listed in the section on the :doc:`hydrodynamically correct model<hyco>`.
+
+.. _bead_bead_theory:
+
+Bead-bead coupling
+^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+    The implementation of the coupling correction models is still alpha functionality.
+    While usable, this has not yet been tested in a large number of different scenarios.
+
+The active calibration method presented in the previous sections relies on oscillating the nanostage with a known amplitude and frequency.
+The fluid in the flow-cell follows the stage motion.
+This in turn exerts a drag on the bead that leads to a sinusoidal displacement of the bead from the trap center.
+The amplitude of the detected displacement (measured in Volts) and the stage amplitude are then quantified.
+From the stage amplitude (measured in microns, since the stage position is calibrated) an expected bead displacement is calculated.
+
+When using two beads, the flow field around the beads is reduced (because the presence of the additional bead slows down the fluid).
+The magnitude of this effect depends on the bead diameter, distance between the beads and their orientation with respect to the fluid flow.
+Streamlines for some bead configurations are shown below (simulated using FEniCSx :cite:`the_fenics_project_developers_2023_10432590`).
+
+.. image:: figures/streamlines.png
+  :nbattach:
+
+As a result, the bead moves less than expected for a given stage motion.
+Since the displacement sensitivity (microns/V) is given by the ratio of the expected bead displacement (in microns) to detected displacement (in Volts) and we detected less displacement than expected (lower voltage amplitude), we obtain an artificially higher displacement sensitivity than expected.
+To correctly take this into account, we need to take into account what happens to the fluid around the beads.
+
+Considering the fluid velocity and viscosity, we can conclude that we typically operate in the regime where viscous effects are dominant (creeping flow).
+This can be checked by calculating the Reynolds number for the flow.
+Filling in the maximal velocity we expect during the oscillation, we find the following expression.
+
+.. math::
+
+    Re = \frac{\rho u L}{\mu} = 2 \pi f A d \frac{\rho}{\mu}
+
+Here :math:`\rho` refers to the fluid density, :math:`u` the characteristic velocity, :math:`L` the characteristic length scale and :math:`\mu` the viscosity.
+For microfluidic flow, this value is typically much smaller than `1`.
```

Comment:
And f, A and d are....

Reply:
These were all defined above. But yeah, it's probably not a bad idea to not assume people having read the preceding section. Makes it nicer to read only this section.

Reply:
ah yes, I see they are defined in the previous section. Yes, good to repeat them here.